.env.test

```diff
@@ -0,0 +1,6 @@
+DB_PASSWORD=password 
```

Comment:
As this file is in the `.gitignore`, it shouldnt be included here. Generally `.env` files containing passwords or API keys should _never_ be committed to a repo as its a security risk in the real world

Reply:
I recognised I did this last night in my book library project. Is it correct that `.gitignore` file should be committed too?

Reply:
Committing gitignore itself is fine yeah